src/main/java/org/plumelib/util/ArraysPlume.java

```diff
@@ -468,7 +468,7 @@ public static double sum(double[][] a) {
    * @see java.util.List#indexOf(java.lang.Object)
    */
   @Pure
-  public static <T> int indexOf(T[] a, @Nullable Object elt) {
+  public static <T> int indexOf(@Nullable T[] a, @Nullable Object elt) {
```

Comment:
What is the purpose of this?  `T` could already be instantiated to a nullable type.  Do you have client code that type-checks with the annotation but not without the annotation?  I have the same question about a similar change below.

Reply:
Yes you are right! For some reason i forgot to check this. It is exactly the same.